### DISCUSSION TITLE
Invalid flag

### DISCUSSION THREAD

**Discussion by MatejaSrejic** [shreyashsaitwal/rush-cli#8] [General]
Seem to be having an issue compiling it... it always seems to return an "invalid flag" error using `rush build -r` or `rush build --release`

```
PS D:\Rush\src\skysheets> rush build --release
                      __
     _______  _______/ /_
    / ___/ / / / ___/ __ \
   / /  / /_/ (__  / / / /
  /_/   \__,_/____/_/ /_/

• Build initialized

┌ Validating project files
│
│  OKAY Metadata file (rush.yml)
│
│  OKAY AndroidManifest.xml file
└ Done
┌ Compiling Java files
│
│  ERR src     error: invalid flag: Studios\AppData\Roaming\rush\workspaces\com.skysheets\classes
│       Usage: javac <options> <source files>
│       use --help for a list of possible options
│
│  Total errors: 1
└ Failed

• Build failed
```

**Comment by MatejaSrejic** [comment]
Sometimes I even just get the following (even with the x flag for extra debug info)

```
• Build initialized

┌ Validating project files
│
│  OKAY Metadata file (rush.yml)
│
│  OKAY AndroidManifest.xml file
└ Done
┌ Compiling Java files
│
│  Total errors: 1
└ Failed

• Build failed
```

**Reply by shreyashsaitwal** [reply]
Can you try building with `-rx` flag and then paste the output here?

**Reply by MatejaSrejic** [reply]
Actually I found the issue. Again, it was a blank space. I saw this mentioned in the AI2 community too but I don't have any spaces in my project name. Guess what, it was the space in my profile name! "c:\users\ont studios\..."

I made a new user on my windows machine and it worked.

I just found out about this.
